### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by sumaerjolly**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    player1selection = player1selection.toUpperCase();
+  }
+  return playerFactory(player1name, player1selection);
+})();
+
+const player2 = (() => {
+  const player2name = prompt('Player 2 please enter your name');// eslint-disable-line no-alert
+  const player2selection = (player1.selection === 'X') ? 'O' : 'X';
+  return playerFactory(player2name, player2selection);
+})();
```
Comment (SunnyAsar) [inline script.js]: let's simplify player1 and player2 with a loop, and do it at game start.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+let boardArray = [0, 1, 2, 3, 4, 5, 6, 7, 8];
```
Comment (SunnyAsar) [inline script.js]: let's also avoid global code.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+
+function putInBoard(id, selection) {
+  boardArray[id] = selection;
+}
+
+function winnerDisplay(winner) {
+  resultDisplay.style.display = 'block';
+  resultText.innerHTML = `The winner is ${winner}`;
+}
```
Comment (SunnyAsar) [inline script.js]: we have multiple methods for displaying message, lets use just one.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @sumaerjolly  and @othman-19  👋 
### Good job on this project, It works!!
To make it better let's work on our code composition, below i have made some comments, lets consider them and make some updates.

**Please request another review when the updates are done.**

**Review by shloch — APPROVED** [review]
### Hello @sumaerjolly  and @othman-19  👋

Great job, please dont forget to include the link of live demo in your README before merging

## This project is considered valid 😊😊😊

**Pull request merged**